src/include/rbd/librbd.h

```diff
@@ -758,6 +768,15 @@ CEPH_RBD_API void rbd_group_image_status_cleanup(
 CEPH_RBD_API void rbd_group_image_status_list_cleanup(
 					      rbd_group_image_status_t *images,
 					      size_t len);
+CEPH_RBD_API int rbd_group_snap_create(rados_ioctx_t group_p, const char *group_name,
+					const char *snap_name);
+CEPH_RBD_API int rbd_group_snap_remove(rados_ioctx_t group_p, const char *group_name,
+					const char *snap_name);
+CEPH_RBD_API int rbd_group_snap_list(rados_ioctx_t group_p, const char *group_name,
+				     rbd_group_snap_spec_t *snaps, size_t *snaps_size);
+CEPH_RBD_API void rbd_group_snap_spec_cleanup(rbd_group_snap_spec_t *group_snap_spec);
```

Comment:
Correct --- I don't see a use-case for cleaning up just one structure vs freeing up the entire list you just retrieved via the _list method.